src/v/kafka/protocol/types.h

```diff
@@ -180,4 +180,15 @@ enum class describe_client_quotas_match_type : int8_t {
 
 std::ostream& operator<<(std::ostream&, describe_client_quotas_match_type t);
 
+/*
+ * The names of group states.
+ */
+static constexpr std::string_view group_state_name_empty = "Empty";
+static constexpr std::string_view group_state_name_preparing_rebalance
+  = "PreparingRebalance";
+static constexpr std::string_view group_state_name_completing_rebalance
+  = "CompletingRebalance";
+static constexpr std::string_view group_state_name_stable = "Stable";
+static constexpr std::string_view group_state_name_dead = "Dead";
```

Comment:
Headers should prefer `inline constexpr` over `static constexpr`

Reply:
ah yeh thanks. fixed